### PR TITLE
[TEMP] common.mk: Disable compressed APEX

### PR DESCRIPTION
**Pull request by Haxk20**
As a temp fix lets disable compressed APEX which causes a12 to not boot on
4.14 kernel.

Please test !!!!!

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -68,6 +68,11 @@ PRODUCT_PACKAGES += \
 # Force building a recovery image: Needed for OTA packaging to work since Q
 PRODUCT_BUILD_RECOVERY_IMAGE := true
 
+ifneq ($(filter 4.14, $(SOMC_KERNEL_VERSION)),)
```
Comment (MarijnS95) [inline common.mk:71]:
```suggestion
ifeq ($(SOMC_KERNEL_VERSION), 4.14)
```

Work smarter, not harder.

**Comment by mdsohail99** [comment]
> As a temp fix lets disable compressed APEX which causes a12 to not boot on 4.14 kernel.
> 
> Please test !!!!!

Hi brother I am noob to developing, can you please tell me or guide me how to do that. Thanks.

**Review by ShujathMohd — CHANGES_REQUESTED** [review]
The flag should be declared immediately/after the inheritance of updatable_apex.mk which is in platform.
By declaring here the flag t is getting overriden/ignored.

**Comment by bartcubbins** [comment]
or we can set it with OVERRIDE_PRODUCT_COMPRESSED_APEX flag

**Comment by MarijnS95** [comment]
`OVERRIDE_PRODUCT_COMPRESSED_APEX` works fine for me, no idea why this wasn't tested.

**Comment by Haxk20** [comment]
Because my pdx201 rebooted once i wiped data. And i didnt get to it yet.
Thus has test please with lot of !

**Comment by MarijnS95** [comment]
> Because my pdx201 rebooted

No need to submit PRs if you know your solution doesn't resolve the issue :)

**Pull request closed without merging**